### PR TITLE
Added our psul global vars

### PR DESCRIPTION
**Pull request by schwent**

### CODE REVIEW

File: scss/base/_globla-variables.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+
+// Paths
+$theme_root: '/sites/all/themes/custom/f5_psul';
```
Comment (whereismyjetpack) [inline scss/base/_globla-variables.scss:4]: this probably isn't used :)

**Comment by whereismyjetpack** [comment]
I think we'll want to add this into `webpack.mix.js` to get compiled at build time.

**Review by whereismyjetpack — CHANGES_REQUESTED** [review]

**Comment by whereismyjetpack** [comment]
mv to legacy scss directory

**Comment by schwent** [comment]
Made some file additions

File: scss/style.scss [hunk 2]
```diff
@@ -1,3 +1 @@
-.test {
-  background-color: red;
-}
+@import "base/*";
```
Comment (whereismyjetpack) [inline scss/style.scss:1]:
```suggestion
@import "base/*";
@import "layout/*";
```

**Comment by schwent** [comment]
@zipymonkey and @whereismyjetpack - this is really old, and I think we've moved way passed this. Should I close it?

**Pull request closed without merging**